recipes/palikiss/build.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+make PREFIX=$PREFIX CC=$CC -C Misc/Applications/pAliKiss all
+make PREFIX=$PREFIX CC=$CC -C Misc/Applications/pAliKiss install-program
```

Comment:
```suggestion
make -j ${CPU_COUNT} PREFIX=$PREFIX CC=$CC -C Misc/Applications/pAliKiss install-program
```

Reply:
as this target is basically copies files, I don't think it needs to run in parallel

Reply:
Then why does it need ` CC=$CC` ?

Reply:
this is to obtain the ARCHTRIPLE https://github.com/jlab/fold-grammars/blob/0ee5347c5d5559b340589338dea25b2ecb8bdfcb/makefile#L18 and https://github.com/jlab/fold-grammars/blob/0ee5347c5d5559b340589338dea25b2ecb8bdfcb/Misc/Applications/RNAshapes/makefile#L22 which is part of a temporary directory storing compile results